index.html

```diff
@@ -10,16 +10,60 @@
-                <a class="btn btn-primary btn-lg" href="javascript:;">Learn more</a>
-            </p>
-            </div>
+
+    <nav class="navbar navbar-expand-sm navbar-light bg-light">
+        <a class="navbar-brand mx-auto">
+        <img src="images/conflist.svg" alt="conflist logo">
+        </a>
+
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarTogglerDemo02" aria-controls="navbarTogglerDemo02" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+        </button>
+
+        <div class="collapse navbar-collapse" id="navbarTogglerDemo02">
+
+            <form class="form-inline mt-4">
+                <input class="form-control" type="text" placeholder="Search by location or technology...">
+            </form>
+
+            <ul class="nav navbar-nav justify-content-end ml-auto">
+                <li class="nav-item">
+                    <a class="nav-link " href="#">Home</a>
+                </li>
+
+                <li class="nav-item">
+                    <a class="nav-link" href="#">Suggest a conference</a>
+                </li>
+
+                <li class="nav-item dropdown">
+                    <a class="nav-link dropdown-toggle active" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                    <img src="images/superKalo.jpg" width="30" height="30" alt="profile picture">Kaloyan
+                </a>
+
+                <div class="dropdown-menu">
+                <a class="dropdown-item" href="#">Profile Settings</a>
+                <a class="dropdown-item" href="#">My Subscriptions</a>
+                <a class="dropdown-item" href="#">Wanna go list</a>
+                <a class="dropdown-item" href="#">Log out</a>
+                </div>
+                </li>
+            </ul>
+        </div>
+    </nav>
+
+    <div class="container">
+
+        <div class="jumbotron">
+            <h1>Hello world!</h1>
+            <p>This is a simple hero unit, a simple jumbotron-style component for calling extra attention to featured content or information.</p>
+            <a class="btn btn-primary btn-lg" href="javascript:;">Learn more</a>
+        </p>
         </div>
+
     </div>
+
+<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.11.0/umd/popper.min.js" integrity="sha384-b/U6ypiBEHpOf/4+1nzFpr53nxSS+GLCkfwBdFNTxtclqqenISfwAzpKaMNFNmj4" crossorigin="anonymous"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/js/bootstrap.min.js" integrity="sha384-h0AbiXch4ZDo7tp9hKZ4TsHbi047NrKGLO3SEJAg45jXxnGIfYzk4Si90RDIqNm1" crossorigin="anonymous"></script>
```

Comment:
Hm. We install these dependencies via NPM, so we have them in the `node_modules` directory. Then, why you load them via CDN?